Example/Standard Integration/BrowseProductsViewController.swift

```diff
@@ -77,13 +77,29 @@ class BrowseProductsViewController: UICollectionViewController {
         super.viewDidLoad()
         self.navigationItem.title = "Emoji Apparel"
         self.navigationController?.navigationBar.isTranslucent = false
+        #if canImport(CryptoKit)
+        if #available(iOS 13.0, *) {
+            self.navigationController?.view.backgroundColor = .systemBackground
+        } else {
+            self.navigationController?.view.backgroundColor = .white
+        }
+        #else
         self.navigationController?.view.backgroundColor = .white
```

Comment:
oh maybe you could put these calls unconditionally before `#if canImport(CryptoKit)` and overwrite the property in the `#if` block, to avoid some duplication.

Don't think it matters much though, not a blocker.

Reply:
I thought about that, but I didn't want the extra cycles of calling setBackgroundColor twice. Not sure if it really matters, the calls are probably all coalesced until the redraw.